src/book/03-guides/03-hardware/02-servo-calibration.mdx

```diff
@@ -0,0 +1,109 @@
+
+1. Select "**Scan**" and wait for the attached servo to be discovered.
+
+   Optional: brand new servos should be preset to an ID of 1. Lower the ID range in "**Options > Scan**" to speed up discovery.
+   ![Screenshot of Dynamixel Wizard performing a scan with no servos discovered](./images/servo_calibration/scan.png)
+   ![Screenshot of Dynamixel Wizard performing a scan with 3 servos discovered](./images/servo_calibration/discover.png)
+
+2. If there are multiple servos discovered, the new servo should be the option with ID = 1 and Baud Rate = 57600 bps.
+
+3. Update the following parameters (don't forget to press save after each update):
+
+   - ID: Varies depending on where the servo will be placed. See below for guidance.
+
+     | Servo Position       | ID  |
+     | -------------------- | --- |
+     | Right Shoulder Pitch | 1   |
+     | Left Shoulder Pitch  | 2   |
+     | Right Shoulder Roll  | 3   |
+     | Left Shoulder Roll   | 4   |
+     | Right Elbow          | 5   |
+     | Left Elbow           | 6   |
+     | Right Hip Yaw        | 7   |
+     | Left Hip Yaw         | 8   |
+     | Right Hip Roll       | 9   |
+     | Left Hip Roll        | 10  |
+     | Right Hip Pitch      | 11  |
+     | Left Hip Pitch       | 12  |
+     | Right Knee           | 13  |
+     | Left Knee            | 14  |
+     | Right Ankle Pitch    | 15  |
+     | Left Ankle Pitch     | 16  |
+     | Right Ankle Roll     | 17  |
+     | Left Ankle Roll      | 18  |
+     | Head Yaw             | 19  |
+     | Head Pitch           | 20  |
+
+     Note that roll is about the x-axis, pitch is about the y-axis, and yaw is about the z-axis. The below diagram specifies the position of each servo within the robot when it is in its default (zombie) position.
+     <details>
+        <summary>Detailed diagram of NUgus servo axes</summary>
+
+     ![Diagram of NUgus robot with roll, pitch and yaw axes defined for each servo.](./images/servo_calibration/nugus_servo_axis.svg)
+     </details>
+
+   - Baud Rate (Bus): 1M bps = 2M/(1 + 1)
+
+     ![Screenshot of Dynamixel Wizard Baud Rate selection with 1M bps selected](./images/servo_calibration/update_parameters.png)
+
+## Testing
+
+To test the servo, perform the following instructions:
+
+1. Set the Goal Position in Dynamixel Wizard to 0, +90 and -90 and observe the position of the arrow on the horn. It should match the picture.
+
+2. Set the Goal Position in Dynamixel Wizard to some position either side of +180 to test if the servo is in joint mode. If it is, it should take the long route around to the second position. If it does not, it is not in joint mode.
+   - To convert into joint mode, select "**Joint**" from the dropdown menu for the servo.
+     ![Screenshot of Dynamixel Wizard with mode type highlighted using a red box](./images/servo_calibration/to_joint.png)
+
+If the servo is performing correctly, we can disconnect the servo and use it in a robot.
```

Comment:
Any error that show up during this process (there should be some sort of error indicator in the status bar of Dynamixel Wizard if I am not mistaken and the LED on the servo will remain lit I think .... its been a while :P) should result in the servo being marked as "bad" and sent off for a more thorough diagnostic (by Peter Turner). Before being flaggeed as "bad" re-flashing the firmware and calibration could be performed a second time just to confirm

Reply:
I'm gonna make an educated guess on the error indication since I can't test it in person, let me know if it seems close enough lol.